### ISSUE TITLE
Installation of cvxpy 1.1.1 for python 3.7 

### ISSUE THREAD

**Issue by LuigiSimeone** [cvxgrp/cvxpy#1049]
**Describe the bug**
Hi,
I am having serious issues trying to install cvxpy. I use Jupiter NB.
I tried many things..
pip install cvxpy fails, 

I tried installed eco and launch it again and still fails, then I also downloaded the .egg of cvxpy 1.1.1 from pypi and put it into the C:\Users\Notebook\anaconda3\Lib\site-packages but nothing..
It is 1 day I try to fix this and havent got a clue
My Python version is 3.7.6
The jupiter version is 6.0.3
My OS is windows 10
When I run pip install cvxpy takes ages and never shows anything, keeps running forever or at least after like 2 hours I kill the kernel

I have read around that can be solved by dowgrading the python version but I do not want to do that.
Please can you help me?
Kind regards
Luigi

**Comment by akshayka** [comment]
Luigi, sorry that you're having trouble.

A Python 3.7 Windows wheel for CVXPY 1.1.1 is available on PyPi. 

CVXPY 1.1 is also available on conda forge (again, for Windows, Python 3.7): https://anaconda.org/conda-forge/cvxpy/files.

Installation should never take hours, it should only take a few minutes at most, so something is wrong, perhaps with your environment.

I would recommend trying the following: 

(1) Create a fresh (new) conda environment, without any packages installed. Then try obtaining CVXPY 1.1.1 from conda forge.

If that fails, try

(2) Create a new (empty) virtual environment (virtualenv), and make sure not to nest this environment in a conda environment. Then try obtaining CVXPY 1.1.1 from PyPI.

If both of those things don't work, you can report back here. One thing --- if those don't work, please post the full print-out from the installation process (or attach it as a Github gist), so others can help figure out what's wrong.

**Comment by akshayka** [comment]
Here are some more details on how to do (1):

Open the Anaconda prompt.
Create a new environment: conda create --name cvxpy
Activate it: conda activate cvxpy
Install CVXPY from conda forge: conda install -c conda-forge cvxpy

**Comment by LuigiSimeone** [comment]
Hi Akshayka,
thanks for your prompt answer.
In the end I was able to solve it.
I switched to conda-forge and then install cvxpy and it works.
Thanks
Luigi

**Comment by LuigiSimeone** [comment]
Hi,
I have encoutered a strange issue.
As I closed the previous issue all was working fine.
I was able to run optimisation with cvxpy.
However I passed the code to a colleague to run it.
I have python 3.7.6 he has 3.6.x.

I ask him to install the cvxpy as conda install -c conda-forge cvxpy
he did it all was going well, but when he runs the code it gets 
cvxpy.error.DCPerror: problem does not follow DCP rules- Specifically the DCP is not DCP: Its following subexpressions are not:
[......] and here a list of vectors/arrays

Now how is that possible that with same code, same package cvxpy he gets the error and I do not get it?
many thanks for helping
